### PR TITLE
Tickets/dm 42883

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: python/lsst/ts/mtrotator/rotator_csc.py [hunk 1]
```diff
@@ -245,12 +245,12 @@ async def check_ccw_following_error(self) -> None:
...
-            rot_tai = rot_data.timestamp
-
             # Get camera cable wrap telemetry and check its age.
-            ccw_data = self.mtmount_remote.tel_cameraCableWrap.get()
-            if ccw_data is None:
+            try:
+                ccw_data = await self.mtmount_remote.tel_cameraCableWrap.next(
+                    flush=True, timeout=MAX_CCW_TELEMETRY_AGE
+                )
+            except asyncio.TimeoutError:
```
Comment (teweitsai) [inline python/lsst/ts/mtrotator/rotator_csc.py:253]: What will happen in the edge case that the `ccw_data` is `None` (no data), and you call the `flush=True`? Will it throw out an error different from `asyncio.TimeoutError` ?

File: tests/test_csc.py [hunk 2]
```diff
@@ -401,19 +421,25 @@ async def test_transient_excessive_ccw_following_error(self) -> None:
...
+            while self.ccw_transient_following_error != 0:
+                await asyncio.sleep(WAIT_FOR_CCW_DELAY)
             # Set the # of fails to 1 and try again;
             # this time a single transient should cause failure.
             self.csc.config.num_ccw_following_errors = 1
             self.ccw_transient_following_error = (
                 self.csc.config.max_ccw_following_error + 0.1
             )
-            await self.assert_next_summary_state(salobj.State.FAULT)
+            print("Waiting for CSC to go to FAULT.")
```
Comment (teweitsai) [inline tests/test_csc.py:439]: Based on your update, maybe `self.log.debug()` instead of `print()`?

**Review by teweitsai — APPROVED** [review]
Looks good
